index.bs

```diff
@@ -1240,13 +1240,13 @@ would look like
 
 <pre><code class="lang-javascript">
   class ReadableStreamDefaultReader {
-    constructor(stream)
+    <a href="#default-reader-constructor">constructor(stream)</a>
```

Comment:
This is desirable for readability of the output, but it sure makes the source unreadable.

@tabatkins Can Bikeshed help our markup look nicer here?

Reply:
I had a different markup in #872 but the consensus was that simple a tags were preferred. I think these links could be automatically generated, so I am in favor of somehow using bikeshed for this.

Reply:
If we end up doing what @ricea suggested in #907 we'd align more with existing specs so I wouldn't suggest adding anything to Bikeshed for this definition.

That said, the domintro blocks (as proposed in #907) are also not very readable in the source either. See e.g. https://github.com/domenic/async-local-storage/blob/master/spec.bs#L239-L249 for my latest attempt. (Explicit `<p>`s needed because of https://github.com/tabatkins/bikeshed/issues/1203.)

Personally I'm not too worried about the source readability here.